exporter/awsxrayexporter/internal/translator/segment_test.go

```diff
@@ -62,13 +62,15 @@ func TestClientSpanWithRpcAwsSdkClientAttributes(t *testing.T) {
 	segment, _ := MakeSegment(span, resource, nil, false)
 	assert.Equal(t, "DynamoDB", *segment.Name)
 	assert.Equal(t, conventions.AttributeCloudProviderAWS, *segment.Namespace)
+	assert.Equal(t, "GetItem", *segment.AWS.Operation)
```

Comment:
Can we also add the case to `aws_test.go`?

Reply:
Probably the rpc.method version of https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awsxrayexporter/internal/translator/aws_test.go#L256

Reply:
I think that's an excellent suggestion! I added a test case like this one just to test that `rpc.method` is read:

```go
func TestAwsWithRpcAttributes(t *testing.T) {
	resource := pdata.NewResource()
	attributes := make(map[string]pdata.AttributeValue)
	attributes[conventions.AttributeRPCMethod] = pdata.NewAttributeValueString("ListBuckets")

	_, awsData := makeAws(attributes, resource)

	assert.NotNil(t, awsData)
	assert.Equal(t, "ListBuckets", *awsData.Operation)
}
```

and added this following above the line you linked to test the case where they are both defined:

```go
attributes[conventions.AttributeRPCMethod] = pdata.NewAttributeValueString("IncorrectAWSSDKOperation")
```